src/lesson14/classwork/table_n_entries.sql

```diff
@@ -0,0 +1,30 @@
+SELECT *
+FROM products
+WHERE product_id = ANY(
+    SELECT product_id
+    FROM (
+            SELECT product_id, unit_price,
+                ROW_NUMBER() OVER(ORDER BY unit_price DESC) AS nth
+            FROM products
+        )sorted_prices
+    WHERE nth = 4
+    );
+
+SELECT *
+FROM (
+        SELECT product_id, product_name, category_id, unit_price, units_in_stock,
```

Comment:
It would better to keep this style of writing sql code:
SELECT
product_id, 
product_name,
category_id, 
unit_price, 
units_in_stock,
...